modules/rostests/apitests/win32nt/ntuser/NtUserCreateAcceleratorTable.c

```diff
@@ -0,0 +1,32 @@
+/*
+ * PROJECT:     ReactOS api tests
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Test for NtUserCreateAcceleratorTable
+ * COPYRIGHT:   Copyright 2025 Max Korostil (mrmks04@yandex.ru)
+ */
+
+#include "../win32nt.h"
+
+
+START_TEST(NtUserCreateAcceleratorTable)
+{
+    HACCEL hAccel;
+    ACCEL Entries[5];
+    ULONG EntriesCount = 0x80000005;
+
+    hAccel = NtUserCreateAcceleratorTable(Entries, EntriesCount);
```

Comment:
I wonder whether, for these sort of tests, you should surround the calls in SEH.
(See https://git.reactos.org/?p=reactos.git;a=blob;f=modules/rostests/apitests/win32nt/ntuser/NtUserGetKeyboardLayoutName.c;h=af8af7c44b45249fea9018eb2aca8d0d1df6ec52;hb=HEAD for an example: this requires `#include <pseh/pseh2.h>` at the top, and the tested code inside a `_SEH2_TRY { ... } _SEH2_EXCEPT(EXCEPTION_EXECUTE_HANDLER) { ... } _SEH2_END;` block.)

Reply:
Add try/except blocks.